docs/using-hive.md

```diff
@@ -237,11 +237,10 @@ To use this feature:
          name: hive
        spec:
          managedDomains:
-         - hive.example.com
-         externalDNS:
-           aws:
-             credentialsSecretRef:
-               name: route53-aws-creds
+         - credentialsSecretRef:
```

Comment:
This is not the correct format. It should be the following.

```
managedDomains:
- aws:
    credentialsSecretRef:
      name: route53-aws-creds
  domains:
  - hive.example.com
```
```

Reply:
+1 to the above, my entry fails to write the cloudprovider as the manged-domain in the config-map.